tutorials/3d/baked_lightmaps.rst

```diff
@@ -3,43 +3,70 @@
 Baked lightmaps
 ===============
```

Comment:
Sounds good, but I'll rename the page to *Using LightmapGI* to avoid confusion with *LightmapGI* in the class reference. (This is why some documentation pages are prefixed with "Using".)